app/src/app/content-plugin/store/effects/upload.effects.ts

```diff
@@ -196,4 +198,36 @@ export class UploadEffects {
       });
     });
   }
+
+  private registerFocusingCreateMenuButton(input: HTMLInputElement): void {
+    input.addEventListener(
+      'click',
+      () => {
+        window.addEventListener(
+          'focus',
+          () => {
+            const createMenuButton = document.querySelector<HTMLElement>('app-create-menu button');
+            const oldBackgroundColor = createMenuButton.style.backgroundColor;
+            createMenuButton.style.backgroundColor = 'rgb(221, 221, 221)';
```

Comment:
this color is hardcoded and won't work with different themes

Reply:
I have changed approach a bit and pushed change.